drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/ConditionColumnSynchronizerTest.java

```diff
@@ -827,7 +912,7 @@ public void testDelete3() throws ModelSynchronizer.MoveColumnVetoException {
     @Test
     public void testMoveColumnTo_MoveLeft() throws ModelSynchronizer.MoveColumnVetoException {
```

Comment:
Would it not have been better to create a new test rather than change this one from using a bound name to negated? (Or have I missed something more subtle?)